scalding-parquet/src/main/scala/com/twitter/scalding/parquet/tuple/scheme/TypedParquetTupleScheme.scala

```diff
@@ -102,7 +125,7 @@ trait ParquetWriteSupport[T] extends WriteSupport[T] {
  * @param fp filter predicate
  * @tparam T tuple value type
  */
-class TypedParquetTupleScheme[T](val readSupport: Class[_], val writeSupport: Class[_],
+class TypedParquetTupleScheme[T](val readSupport: ParquetReadSupport[T], val writeSupport: ParquetWriteSupport[T],
```

Comment:
these might not be serializable. You might want to use `@transient` here and `com.twitter.chill.Externalizer` which attempts both Java and Kryo serialization. Cascading needs schemes to be serializable, I think.